src/components/Proudects.js

```diff
@@ -0,0 +1,34 @@
+/* eslint-disable jsx-a11y/alt-text */
+import React, { useEffect, useState,useContext } from 'react'
+import {ProductContxt} from '../App'
+import axios from 'axios'
+import '../style/Products.css'
+function Proudects() {
+    const [products, SetData] = useState();
```

Comment:
It's better to be consistent in the getter and setter of state. 
`    const [products, SetProducts] = useState();
`

Reply:
done